### PR TITLE
Bridge docs

### PR DESCRIPTION
**Pull request by sukantoraymond**
# Docs PR Template

## Why this should be merged

<please give a thorough description of why these changes are necessary>

## How this works

<please give a thorough description of what your changes are and how they work>

## How these changes were tested 

<please give a thorough description of how to view the changes and how they were tested>

## To prevent any errors while building

- [x] run `vale /docs/file/path` on all changed `.md` files to ensure all grammar rules pass
- [x] run `markdownlint /docs/file/path` on all changed `.md` files to ensure all linting rules pass
- [x] complete the above two checks and all additional rules outlined in `style-checker-notes.md` to
  ensure all checks pass

### If a document was removed/deleted

- [x] the path to that doc must be redirected to a valid URL via the
  `_redirects` file
- [x] `_redirects` were manually verified with the cloudflare preview link
- [x] `sidebars.json` reflects all changes made to file path

### If a document was moved

- [x] all files that were moved from their current directory to a new path have had their paths
  redirected via the `_redirects` file
- [x] `_redirects` were manually verified with the cloudflare preview link
- [x] `sidebars.json` reflects all changes made to file path

### CODE REVIEW

File: docs/tooling/cli-cross-chain/teleporter-token-bridge-on-local-networks.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+- To create a Teleporter-enabled Subnet configuration, [visit here](/tooling/cli-cross-chain/teleporter-on-local-networks.md#create-subnet-configurations)
+- To deploy a Teleporter-enabled Subnet, [visit here](/tooling/cli-cross-chain/teleporter-on-local-networks.md#deploy-the-subnets-to-local-network)
+
+## Deploy ERC-20 Token in C-Chain
+
+First, let's create an ERC-20 Token and deploy it to C-Chain. For our example, it will be called 
+TOK. 
+
+Sample script to deploy the ERC-20 Token can be found [here](https://github.com/ava-labs/avalanche-cli/tree/add-deploy-script/deploy_erc20).
```
Comment (felipemadero) [inline docs/tooling/cli-cross-chain/teleporter-token-bridge-on-local-networks.md:38]: some points:
1) add-deploy-script it should live under examples, not directly under avalanche-cli root
2) probably the rename should be from `add-deploy-script/deploy-erc20` to `examples/erc20-deploy`
3) we don't have this into CLI main

File: docs/tooling/cli-cross-chain/teleporter-token-bridge-on-local-networks.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+- To create a Teleporter-enabled Subnet configuration, [visit here](/tooling/cli-cross-chain/teleporter-on-local-networks.md#create-subnet-configurations)
+- To deploy a Teleporter-enabled Subnet, [visit here](/tooling/cli-cross-chain/teleporter-on-local-networks.md#deploy-the-subnets-to-local-network)
+
+## Deploy ERC-20 Token in C-Chain
+
+First, let's create an ERC-20 Token and deploy it to C-Chain. For our example, it will be called 
+TOK. 
+
+Sample script to deploy the ERC-20 Token can be found [here](https://github.com/ava-labs/avalanche-cli/tree/add-deploy-script/deploy_erc20). 
+Note that you will need to modify the script [here](https://github.com/ava-labs/avalanche-cli/blob/add-deploy-script/deploy_erc20/deploy_erc20.sh#L17)
```
Comment (felipemadero) [inline docs/tooling/cli-cross-chain/teleporter-token-bridge-on-local-networks.md:39]: is that true? why the script needs to be modified? it is assumed to live in the same directory where remappings.txt and lib live

File: docs/tooling/cli-cross-chain/teleporter-token-bridge-on-local-networks.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+Remote Deployed to http://127.0.0.1:9650/ext/bc/2TnSWd7odhkDWKYFDZHqU7CvtY8G6m46gWxUnhJRNYu4bznrrc/rpc
+Remote Address: 0x7DD1190e6F6CE8eE13C08F007FdAEE2f881B45D0
+```
+
+Before we transfer our ERC-20 token from C-Chain to our Subnet, we will first call `avalanche key
+list` command to check our initial balances in C-Chain and Subnet. 
+
+We will inquire the balances of our ERC-20 Token TOK both in C-Chain and our Subnet, which has the 
+address of `0x5DB9A7629912EBF95876228C24A848de0bfB43A9` in the C-Chain and address of
+`0x7DD1190e6F6CE8eE13C08F007FdAEE2f881B45D0` in our Remote contract.
```
Comment (felipemadero) [inline docs/tooling/cli-cross-chain/teleporter-token-bridge-on-local-networks.md:94]: `in our Remote Contract` -> `in testSubnet`

File: docs/tooling/cli-cross-chain/teleporter-token-bridge-on-local-networks.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+|        | subnet_testSubnet  | testSubnet | 0x5a4601D594Aa3848cA5EE0770b7883d3DBC666f6 | TOK (0x7DD1.) |               0 | Local Network |
++        + _airdrop           +---------+--------------------------------------------+---------------+-----------------+---------------+
+|        |                    | C-Chain    | 0x5a4601D594Aa3848cA5EE0770b7883d3DBC666f6 | TOK (0x5DB9.) |               0 | Local Network |
++--------+--------------------+---------+--------------------------------------------+---------------+-----------------+---------------+
+```
+
+## Transfer The Token From C-Chain To Our Subnet
+
+Now we will transfer 100 TOK tokens from our ewoq address in C-Chain to subnet_testSubnet_airdrop
+address in our Subnet `testSubnet`. Note that we will be using the Home address `0x4Ac1d98D9cEF99EC6546dEd4Bd550b0b287aaD6D`
```
Comment (felipemadero) [inline docs/tooling/cli-cross-chain/teleporter-token-bridge-on-local-networks.md:114]: `Home address` -> `Home Contract address`

File: docs/tooling/cli-cross-chain/teleporter-token-bridge-on-local-networks.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+and Remote address `0x7DD1190e6F6CE8eE13C08F007FdAEE2f881B45D0`.
```
Comment (felipemadero) [inline docs/tooling/cli-cross-chain/teleporter-token-bridge-on-local-networks.md:115]: `Remote address` -> `Remote contract address`

File: docs/tooling/cli-cross-chain/teleporter-token-bridge-on-local-networks.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+list` command to check our initial balances in C-Chain and Subnet. 
+
+We will inquire the balances of our ERC-20 Token TOK both in C-Chain and our Subnet, which has the 
+address of `0x5DB9A7629912EBF95876228C24A848de0bfB43A9` in the C-Chain and address of
+`0x7DD1190e6F6CE8eE13C08F007FdAEE2f881B45D0` in our Remote contract.
+
+```bash
+avalanche key list --local --keys ewoq,subnet_testSubnet_airdrop --subnets c,testSubnet --tokens 0x5DB9A7629912EBF95876228C24A848de0bfB43A9,0x7DD1190e6F6CE8eE13C08F007FdAEE2f881B45D0
++--------+--------------------+---------+--------------------------------------------+---------------+-----------------+---------------+
+|  KIND  |        NAME        | SUBNET     |                  ADDRESS                   |     TOKEN     |     BALANCE     |    NETWORK    |
```
Comment (felipemadero) [inline docs/tooling/cli-cross-chain/teleporter-token-bridge-on-local-networks.md:99]: there is misalinment along the column separator

File: docs/tooling/cli-cross-chain/teleporter-token-bridge-on-local-networks.md [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+
+We will call `avalanche key list` command again to verify that the transfer is successful.
+
+`subnet_testSubnet_airdrop` should now have 100 TOK tokens in our Subnet `testSubnet` and our EWOQ
+account now has 99900 TOK tokens in C-Chain.
+
+```bash
+avalanche key list --local --keys ewoq,subnet_testSubnet_airdrop --subnets c,testSubnet --tokens 0x5DB9A7629912EBF95876228C24A848de0bfB43A9,0x7DD1190e6F6CE8eE13C08F007FdAEE2f881B45D0
++--------+--------------------+---------+--------------------------------------------+---------------+-----------------+---------------+
+|  KIND  |        NAME        | SUBNET     |                  ADDRESS                   |     TOKEN     |     BALANCE     |    NETWORK    |
```
Comment (felipemadero) [inline docs/tooling/cli-cross-chain/teleporter-token-bridge-on-local-networks.md:140]: there is misalignment

**Review by laviniat1996 — APPROVED** [review]

**Pull request merged**